src/lib.rs

```diff
@@ -456,7 +456,7 @@ impl<'de> Deserialize<'de> for RetryOpts {
     {
         struct RetryOptsVisitor;
 
-        impl<'de> Visitor<'de> for RetryOptsVisitor {
+        impl Visitor<'_> for RetryOptsVisitor {
```

Comment:
This is needed in order to resolve a clippy error.

https://github.com/film42/sidekiq-rs/actions/runs/12557608845/job/35010511817